frontend/src/containers/DonatePage.js

```diff
@@ -238,6 +231,6 @@ function mapStateToProps({
     saveInProgress: users.updateInProgress,
     isAuthenticated: session.isAuthenticated,
     newUser,
-    i18n
+    i18n: i18n(group.settings.lang)
```

Comment:
Why not passing `lang` instead of `i18n`? Looks a bit strange to me to pass a whole library, a lib is not a prop.